tests/Models/DocumentVersion.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Laravel\Tests\Models;
+
+use MongoDB\Laravel\Eloquent\HasDocumentVersion;
+use MongoDB\Laravel\Eloquent\Model as Eloquent;
+
+/** @property int __v */
```

Comment:
This trait should contain a method `getDocumentVersion`. So that you don't have to access the property and worry about the name configuration.